test/features/parsoid/transform.js

```diff
@@ -76,22 +76,41 @@ function findSpecs() {
 }
 
 function x2y(spec) {
-    function test() {
-        return preq.post({
-            uri: server.config.baseURL + '/transform/' + spec.from.format + '/to/' + spec.to.format,
-            headers: { 'content-type': 'application/json' },
-            body: readFile(spec.from.src)
-        })
-        .then(function(res) {
-            assert.deepEqual(res.status, 200);
-            assert.deepEqual(res.body, readFile(spec.to.src));
-        });
+
+    function test(bodyOnly) {
+        return function () {
+            var uri = server.config.baseURL + '/transform/' + spec.from.format +
```

Comment:
Since this is a POST, why don't we set the bodyOnly flag in the POST body / JSON?